config/prod.exs

```diff
@@ -11,8 +11,10 @@ use Mix.Config
 # before starting your production server.
 config :codestories_review_bot, CodestoriesReviewBotWeb.Endpoint,
   http: [:inet6, port: System.get_env("PORT") || 4000],
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  url: [scheme: "https", host: "codestories-review-bot.herokuapp.com", port: System.get_env("PORT")],
+  force_ssl: [rewrite_on: [:x_forwarded_proto]],
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  secret_key_base: Map.fetch!(System.get_env(), "SECRET_KEY_BASE")
```

Comment:
You can use e.g. `secret_key_base: {:system, :string, "SECRET_KEY_BASE"}` syntax in configs (you can also add 4 element to the tuple with default value)